api_tests/lib/actor.ts

```diff
@@ -106,17 +106,44 @@ export class Actor {
         if (network != "regtest") {
             throw Error("Expected network regtest, found " + network);
         }
-        let type = action.type;
-
-        switch (type) {
+        switch (action.type) {
             case "bitcoin-send-amount-to-address": {
                 action.payload.should.include.all.keys("to", "amount");
                 let { to, amount } = action.payload;
 
                 return this.wallet.btc().sendToAddress(to, parseInt(amount));
             }
             case "bitcoin-broadcast-signed-transaction": {
-                let { hex } = action.payload;
+                action.payload.should.include.all.keys("hex", "locktime");
+
+                let fetchMedianTime = async () => {
+                    let blockchainInfo = await bitcoin.getBlockchainInfo();
+                    return blockchainInfo.mediantime;
+                };
+
+                let { hex, locktime } = action.payload;
+
+                // If it is equal, it is rejected by bitcoin-core :(
+                locktime += 1;
```

Comment:
❓ Is bitcoin-core wrong to reject it? I can't remember wether the locktime had to be strictly greater than or could be equal in consensus. If bitcoin-core is right then the action we create should already have +1.